tools/cross/install_rootfs.sh

```diff
@@ -5,7 +5,7 @@ usage()
 {
     echo "Usage: $0 [BuildArch] [LinuxCodeName] [--setproxy=IP] [--skipunmount]"
     echo "BuildArch can be: arm(default), aarch64 and armel"
-    echo "LinuxCodeName - optional, Code name for Linux, can be: bionic(default), trusty, xenial, focal"
+    echo "LinuxCodeName - optional, Code name for Linux, can be: bionic(default), xenial, focal"
```

Comment:
It's trivial, but how about changing the order to xenial, bionic(default), focal?
Anyway, LGTM regardless of it.

Reply:
OK. I'll update it.